### PR TITLE
:sparkles: Add Cast Page

### PR DESCRIPTION
**Pull request by daebert**
globals CSS not working,
also: when we set a background color in our module.css, the image disappears

### CODE REVIEW

File: src/app/components/MovieCard/MovieCard.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+type CardProps = {
+  imgSrc: string;
+};
+
+function MovieCard({ imgSrc }: CardProps): JSX.Element {
+  return (
+    <div>
+      <img className={styles.image} src={imgSrc} alt="" />
+    </div>
```
Comment (phlgr) [inline src/app/components/MovieCard/MovieCard.tsx:12]:
```suggestion
      <img className={styles.image} src={imgSrc} alt="" />
```

File: src/app/pages/Cast/Cast.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import SearchIcon from '../../components/Icons/SearchIcon';
+import MovieCard from '../../components/MovieCard/MovieCard';
+import NavBarLink from '../../components/NavBarLink/NavBarLink';
+import styles from './Cast.module.css';
+
+function Cast(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <BackButton></BackButton>
```
Comment (phlgr) [inline src/app/pages/Cast/Cast.tsx:16]:
```suggestion
        <BackButton />
```

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/components/MovieCard/MovieCard.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import styles from './MovieCard.module.css';
+
+type CardProps = {
+  imgSrc: string;
+};
+
+function MovieCard({ imgSrc }: CardProps): JSX.Element {
+  return (
+    <div>
+      <img className={styles.image} src={imgSrc} alt="" />
```
Comment (lmachens) [inline src/app/components/MovieCard/MovieCard.tsx:11]: An `alt` could make sense here

File: src/app/pages/Cast/Cast.module.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  grid-gap: 1em;
+}
+
+.main {
+  overflow: scroll;
+}
+
+.main::-webkit-scrollbar {
+  display: none;
+}
```
Comment (lmachens) [inline src/app/pages/Cast/Cast.module.css:14]: Why is the scrollbar not displayed?

**Review by lmachens — COMMENTED** [review]
（*＾-＾*）
Could you link the related issue?

**Review by tmh9x — APPROVED** [review]

File: src/app/pages/Cast/Cast.module.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  display: grid;
+  column-gap: 1em;
+  width: 100%;
+  grid-auto-flow: column;
+  overflow: overlay;
+}
+
+.navBar {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (MaxMuecket) [inline src/app/pages/Cast/Cast.module.css:39]:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```

**Review by MaxMuecket — CHANGES_REQUESTED** [review]
Looks good. Just a small change 😉
From BR 2

**Review by Dev-dfm — COMMENTED** [review]
lgtm 🚀

File: src/app/pages/Cast/Cast.module.css (hunk 4 above)
Comment (daebert) [reply src/app/pages/Cast/Cast.module.css:14]: To us, it seems very common on mobile-first applications to not have a scrollbar. The user will see that it is scrollable from the design of the page.

**Comment by daebert** [comment]
> （_＾-＾_）
> Could you link the related issue?

Checked it intitally. Unfortunmately there is no issue for this topic.

**Pull request merged**